packages/contracts-bedrock/tasks/validate-spacers.ts

```diff
@@ -12,10 +16,37 @@ task(
       continue
     }
 
+    // Some files may not have buildInfo (certain libraries). We can safely skip these because we
+    // make sure that everything is accounted for anyway.
     const buildInfo = await hre.artifacts.getBuildInfo(name)
+    if (buildInfo === undefined) {
+      continue
+    }
+
     for (const source of Object.values(buildInfo.output.contracts)) {
       for (const [contractName, contract] of Object.entries(source)) {
         const storageLayout = (contract as any).storageLayout
+
+        // Check that the layout lock is respected.
+        if (layoutLock[contractName]) {
+          const removed = layoutLock[contractName].filter((variable: any) => {
```

Comment:
This is kind of hard to read, very nested

Reply:
Updated